frontend/src/component/menu/routes.ts

```diff
@@ -125,6 +125,7 @@ export const routes: IRoute[] = [
         component: ArchiveProjectList,
         type: 'protected',
         menu: {},
+        enterprise: true,
```

Comment:
Even though this is called `enterprise` I believe (and validated locally that) it does what we want: Only mount this route on Pro and Enterprise.